packages/table/src/TableContext/TableContext.types.ts

```diff
@@ -31,20 +16,20 @@ export interface BaseTableContextValue<T extends LGRowData> {
    * Whether rows in the table are selectable
    */
   isSelectable?: boolean;
+
+  /**
+   * Available [properties and methods](https://tanstack.com/virtual/latest/docs/api/virtualizer#virtualizer-instance) returned from the Virtualizer instance.
+   */
+  virtualTable?: LeafyGreenVirtualTable<T>['virtual'];
```

Comment:
I thought we didn't want to link out to TanStack docs?

Reply:
i'll sort everything out in my documentation PR